jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -70,15 +70,15 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmp_path):
     mm._obj.vue_calculate_moment()
 
     assert mm._obj.moment_available
-    assert dc[1].label == 'moment 0'
+    assert dc[-1].label == 'moment 0'
     mv_data = cubeviz_helper.app.get_viewer(
         cubeviz_helper._default_uncert_viewer_reference_name
     ).data()
     # by default, will overwrite the previous entry (so only one data entry)
     assert len(mv_data) == 1
     assert mv_data[0].label == 'moment 0'
 
-    assert len(dc.links) == 14
+    assert len(dc.links) == 19
```

Comment:
Why are there 5 more links now?

Reply:
There is now a new 1d spectrum loaded and linked when first loading the new flux cube (so probably 3 links) and then the moment map is linking (2 new links).  The _added_ links are:

```
Wave <- forwards(Wave), Pixel Axis 0 [x] <- world2pix(Wave)
Pixel Axis 0 [x] <- world2pix(Wave)
mask <-> flux
Wave <- pix2world(Pixel Axis 0 [x])
Wave <- backwards(Wave)
```

I definitely think this could use some thinking and scrutiny (especially by @javerbukh) to see if we're overlinking anywhere.  With this many changes, I'll admit I was mostly just trying to update tests to pass as long as the changes seemed reasonable, but we should be careful and make sure they actually make sense.

Reply:
Is there a way to find out which data are associated with those links?

Reply:
Plugin request for 4.0: Link viewer. My guess is that the glue-collapsed spectra had fewer links then the jdaviz-collapsed spectra, so 5 more links seems reasonable.

Reply:
Link viewer would be so nice. We already have a ticket for that a long time ago, I think.